### ISSUE TITLE
❌ Seeking developers to fix Elementor's accessibility shortcomings (PAID)

### ISSUE THREAD

**Issue by besweeet** [elementor/elementor#13348]
Elementor is great for many things, but one area it lacks in is regarding accessibility. Here are the accessibility bottlenecks I’ve experienced:

1. Using link anchors (to scroll to another area) doesn’t change keyboard focus
2. Accordions don’t announce expanded / collapsed or selected states
3. Cannot specify a single **main** tag on a page
4. Main menus with sub-menus don’t announce proper roles and values

Our company is taking accessibility seriously to where we are willing to pay someone to help fix these issues. Perhaps the fixes can be turned into a public plugin that sits on top of the base Elementor installation.

**Comment by besweeet** [comment]
Bump... Surely someone can manually override whatever Elementor does?

**Comment by Bovelett** [comment]
hey @besweeet Have you found someone yet?

**Comment by besweeet** [comment]
@Bovelett Unfortunately, I was unsuccessful at finding someone who was willing to fix Elementor's shortcomings.

**Comment by Bovelett** [comment]
@besweeet I'm sorry to hear that. 😔  If all goes well, then by Tuesday I am publishing my new site, which is mainly about my consultancy services for online accessibility. It will be an "in public" project. Meaning that this is mentioned on the site, and that I will disclose how it's built in Elementor. In blog updates (shared on social media) I will speak about:

1. which parts cannot be made accessible by myself (and which GitHub feature requests and bug reports correspond);
2. which parts caused me to have to jump through hoops (and also which GitHub feature requests and bug reports correspond); and
3. how I solved certain issues by myself.

If it's of interest to you, updates will be posted on Twitter (Bovelett) or Linkedin.
